modules/nw-metallb-software-components.adoc

```diff
@@ -14,11 +14,18 @@ When the Operator starts an instance of MetalLB, it starts a `controller` deploy
 `controller`::
 The Operator starts the deployment and a single pod.
 When you add a service of type `LoadBalancer`, Kubernetes uses the `controller` to allocate an IP address from an address pool.
+Ensure you have the following entry in your `controller` pod logs:
```

Comment:
@xenolinux Could you amend this statement as "Ensure you have the following entry in your controller pod logs and in case of failure error related to the service".

Controller is responsible for address assignment and speaker is responsible for IP address announcement, if the service does not have IP address customer needs to look at controller logs, if ip address is assigned but service is not working then speaker logs need to examined.

Reply:
Addressed